### PR TITLE
[7601] Install and configure Celery

### PR DESCRIPTION
**Pull request by hklarner**
to do:
- [x] replace old background tasks decorator with new shared task decorator
- ~custom migration that removes the existing tables of the un-installed app `background_tasks`~

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -16,3 +16,5 @@ python-dateutil==2.8.2
 python-magic==0.4.27
 rules==3.3
 XlsxWriter==3.1.2
+celery==5.3.1
+redis==5.0.0
```
Comment (goapunk) [inline requirements/base.txt:20]: we might not need redis here as a4 doesn't use it / is transcient to which celery broker is used. The actual celery config will be in a+/mb, so maybe we just add it there?

Comment (hklarner) [reply requirements/base.txt]: done

**Review by m4ra — APPROVED** [review]

**Comment by m4ra** [comment]
@hklarner can you also add the changelog?

**Comment by m4ra** [comment]
We said the coverage drop is very little, so we are good to merge.

**Review by goapunk — APPROVED** [review]
LGTM

**Review by m4ra — APPROVED** [review]

**Pull request merged**